modules/hub/pages/manual-mode.adoc

```diff
@@ -0,0 +1,58 @@
+[[manual-mode-example]]
+= Manual mode
+
+
+In the Manual mode the user needs to provide credentials for each {productname} server the user wants to connect to.
+
+Here in this example, following is happening
+
+[square]
+* User provides credentials for the hub to the login method, a `sessionkey` for the hub is returned
+* Using the sessionkey from the previous step, `serverIds` for the servers attached to the hub are requested by calling `listServerIds` method
+* User provides the credentials for each {productname} server, she needs data from and calls the hub method `attachToServers`. Authentication part is done here.
+* A `multicast` call is performed on the specific servers where `serverId` is used to identify the specific sever. Behind the scenes, `system.list_system method` of each {productname} server's XMLRPC API method will be called to get the list of the clients registered with each {productname} server
+* Hub will aggregate the results and return the response in form of a `map`. This map has two entries:
+- `Successful`: contains a list of responses for those clients where the call was successful.
+- `Failed`: contains a list of responses for those clients where the call failed.
+
+[NOTE]
+====
+If user wish to get information from only one {productname} server, then Hub API also provided a convenient `unicast` namespace. In this case, the response would not be a map, instead the response format would be exactly the same as if the user would directly call that {productname} server's API.
+====
+
+* User logs out
+
+
+[source,python]
+----
+#!/usr/bin/python
+import xmlrpclib
+import itertools
+
+HUB_URL = "http://localhost:8888/hub/rpc/api"
+HUB_LOGIN = "admin"
+HUB_PASSWORD = "admin"
+
+client = xmlrpclib.Server(HUB_URL, verbose=0)
+
+hubKey = client.hub.login(HUB_LOGIN, HUB_PASSWORD)
+
+#Get the server Ids
+serverIds = client.hub.listServerIds(hubKey)
+
+# authenticate those servers(each server with it's own credentials)
+usernames = ["admin" for s in serverIds]
+passwords = ["admin" for s in serverIds]
+client.hub.attachToServers(hubKey, serverIds, usernames, passwords)
+
+# perform the needed operation
+systemsPerServer = client.multicast.system.list_systems(hubKey, serverIds)
+successfulResponses = systemsPerServer["Successfull"]["Responses"]
+failedResponses = systemsPerServer["Failed"]["Responses"]
+
+for system in itertools.chain.from_iterable(successfulResponses):
```

Comment:
Is the use of `itertools.chain.from_iterable` really necessary here?

I am most likely missing something - what would make `for response in successfulResponses` not work here?